### PR TITLE
Added Scaled Exponential Linear Unit Activation Function

### LINKED ISSUE #9010: Other Activation Functions
### Feature description

Hi, I'm relatively new to open source and found this repo friendly for early contributions. But is there a reason as to why there are only two activations (ELU and Leaky ReLU), under neural_network / activation_functions.? Also, can I contribute new activations into it?

### PR DESCRIPTION
**Pull request by AdarshAcharya5**
Added Scaled Exponential Linear Unit Activation (SELU) Function under TheAlgorithms/Python/neural_network/activation_functions. Description of SELU taken from reference link provided in the top comment section in scaled_exponential_linear_unit.py
Fixes #9010 

* [X] Add an algorithm?

### Checklist:
* [X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [X] This pull request is all my own work -- I have not plagiarized.
* [X] I know that pull requests will not be merged if they fail the automated tests.
* [X] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [X] All new Python files are placed inside an existing directory.
* [X] All filenames are in all lowercase characters with no spaces or dashes.
* [X] All functions and variable names follow Python naming conventions.
* [X] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [X] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [X] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [X] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

**Comment by AdarshAcharya5** [comment]
@tianyizheng02 Thanks for your reply in the discussion section!. All checks passed now.

File: neural_network/activation_functions/scaled_exponential_linear_unit.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+include Batch-Normalization layers.
+References :
+https://iq.opengenus.org/scaled-exponential-linear-unit/
+"""
+
+import numpy as np
+
+
+def scaled_exponential_linear_unit(
+    vector: np.ndarray, alpha: float = 1.6732, _lambda: float = 1.0507
```
Comment (tianyizheng02) [inline neural_network/activation_functions/scaled_exponential_linear_unit.py:19]: Is there a reason why `_lambda` has an underscore? Is the user not meant to change this coefficient?

Comment (AdarshAcharya5) [reply neural_network/activation_functions/scaled_exponential_linear_unit.py:19]: Yes, firstly lambda is a reserved keyword in python. Secondly both alpha and lambda are fixed constants.

Comment (AdarshAcharya5) [reply neural_network/activation_functions/scaled_exponential_linear_unit.py:19]: The user can change these values, that may yield slightly different behaviour from the function, but the default values given to them are `alpha: float = 1.6732, _lambda: float = 1.0507`.

Comment (AdarshAcharya5) [reply neural_network/activation_functions/scaled_exponential_linear_unit.py:19]: Ah. Yes for sure, I'll get it done right away!

**Review by tianyizheng02 — APPROVED** [review]

**Pull request merged**